src/ets_lru.erl

```diff
@@ -173,10 +179,11 @@ handle_call({insert, Key, Val}, _From, St) ->
             true = ets:delete(St#st.atimes, ATime),
             true = ets:insert(St#st.atimes, {NewATime, Key});
         [] ->
-            Entry = #entry{key=Key, val=Val, atime=NewATime, ctime=NewATime},
+            NewCTime = element(1, NewATime),
```

Comment:
Please avoid magic numbers here. Couple of options:
option 1:
```
line 173:
  {NewCTime, _} = NewATime = strict_monotonic_time(St#st.time_unit),
```

option 2:
```
{NewCTime, _} = NewATime,
```

Reply:
Yes, I think this was a code smell indicating that `ctime` and `atime` should not have different shapes. I've removed it in latest version.